py_neuromodulation/features/bispectra.py

```diff
@@ -127,12 +124,11 @@ def calc_feature(self, data: np.ndarray) -> dict:
             f1s=tuple(self.settings.f1s),  # type: ignore
             f2s=tuple(self.settings.f2s),  # type: ignore
         )
+        waveshape = waveshape.results.get_results(copy=False)  # can overwrite obj with array
```

Comment:
This is a less hacky way of extracting the bispectral info while avoiding a copy, without relying on private attrs.